Zend/tests/exception_004.phpt

```diff
@@ -3,7 +3,7 @@ Throwing exception using a class that isn't derived from the Exception base clas
 --FILE--
 <?php
 
-error_reporting(E_ALL|E_STRICT);
+error_reporting(E_ALL);
```

Comment:
This line can be dropped entirely. (E_ALL is the default error_reporting in tests.)